lib/parsers/acorn.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const acorn = require("acorn");
+const fixParser = require("./acorn-extensions.js").fix;
```

Comment:
Maybe a more descriptive name than `fix`. It looks like the `acorn-extensions` could be a place to host more than 1 extension.